pybamm/expression_tree/unary_operators.py

```diff
@@ -67,6 +67,22 @@ def evaluate(self, t=None, y=None):
         return np.abs(self.children[0].evaluate(t, y))
 
 
+class Function(UnaryOperator):
+    """A node in the expression tree representing an arbitrary function
+
+    **Extends:** :class:`UnaryOperator`
+    """
+
+    def __init__(self, child, func):
```

Comment:
I see your point in the issue about the order of arguments - would switch this one around then, so that we call `FunctionParameter` and `Function` the same way

Reply:
OK - changed that in the new commit